models/application.py

```diff
@@ -67,10 +69,10 @@ def remove(self, name):
     def restart(self, name):
         app = self.get(name)
```

Comment:
`get` switch to throw in case of error, so looks like no need for `if`?

instead if this method is supposed to be safe the `except` block is needed

(and the same below)